### PR TITLE
Update about control

### PR DESCRIPTION
**Pull request by IvenBach**
Users should now have an easier time reporting issues.

### CODE REVIEW

File: Rubberduck.Core/UI/About/AboutControl.xaml [hunk 1]
```diff
@@ -5,112 +5,132 @@
...
+                        Height="100" Width="250"
+                        VerticalAlignment="Center"
+                            MouseLeftButtonDown="CopyVersionInfo_MouseLeftButtonDown">
                 <TextBlock x:Name="Version"
-                           Margin="0,5" Text="{Binding Version}" Foreground="Black"
-                           FontWeight="Bold" FontSize="10"
-                           VerticalAlignment="Center" HorizontalAlignment="Center"/>
+                               Margin="0,5" Text="{Binding Version}" Foreground="Black"
+                               FontWeight="Bold" FontSize="12"
+                               VerticalAlignment="Center" HorizontalAlignment="Center"/>
```
Comment (Vogel612) [inline Rubberduck.Core/UI/About/AboutControl.xaml]: where does the whitespace discrepancy come from?

File: Rubberduck.Core/UI/About/AboutControl.xaml [hunk 2]
```diff
@@ -5,112 +5,132 @@
...
+                       TextAlignment="Center"
+                       Height="100" Width="250"
+                       Background="White">
+                <StackPanel Orientation="Vertical"
+                            HorizontalAlignment="Center"
+                            Margin="10">
+                    <TextBlock HorizontalAlignment="Center"
+                               Text="Found an issue?"/>
+                    <TextBlock HorizontalAlignment="Center"
+                               Text="Help us improve RD!"/>
```
Comment (retailcoder) [inline Rubberduck.Core/UI/About/AboutControl.xaml]: Needs to be a .resx resource string

File: Rubberduck.Core/UI/About/AboutControl.xaml [hunk 3]
```diff
@@ -5,112 +5,132 @@
...
+                                   TextDecorations="{x:Null}">
+                        <Label Height="40"
+                               HorizontalContentAlignment="Center"
+                               VerticalContentAlignment="Center"
+                               Background="#FF40C954"
+                               Margin="10"
+                               Foreground="WhiteSmoke"
+                               Width="120"
+                               FontSize="13"
+                               Content="Report an issue"/>
```
Comment (retailcoder) [inline Rubberduck.Core/UI/About/AboutControl.xaml]: Please move string to resources

File: Rubberduck.Core/UI/About/AboutControl.xaml [hunk 4]
```diff
@@ -5,112 +5,177 @@
...
+                        </Hyperlink>
+                    </TextBlock>
+                    <TextBlock HorizontalAlignment="Center" 
+                               Text="You can also find us at:" />
```
Comment (Vogel612) [inline Rubberduck.Core/UI/About/AboutControl.xaml]: Resx signpost

**Pull request merged**